nebula/ui/components/VPNAlert.qml

```diff
@@ -13,26 +13,26 @@ Rectangle {
     id: alertBox
 
     // Defines what type of alert this is
-    property var alertType: stateNames.info
+    property string alertType: stateNames.info
     // Contains all accepted values for alertType
     readonly property var alertTypes: stateNames
     // Determines if the toast should fit into the layout or put itself on the top
-    property var isLayout: false
+    property bool isLayout: false
```

Comment:
the default value of a QML bool is false, so technically this is redundant BUT I don't think thats totally intuitive and obvious so I think we should set this as a precedent that QML bools should always be defined when they are declared